lib/bootstrap.js

```diff
@@ -0,0 +1,65 @@
+var spawn = require('child_process').spawn;
+var path = require('path');
+var sleep = require('sleep').sleep;
+
+function Bootstrap(codecept, callback) {
+
+  var phantomjs, seleniumPid, callbackCalled = false;
+
+  (function () {
+
+    var protractor = codecept.config.helpers.Protractor;
```

Comment:
this works not only for Protractor but for `WebDriverIO`, `SeleniumWebDriverJS` as well.
probably you should match the first helper with `browser` property in it